app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -67,8 +67,20 @@ def create
     api :GET, "/content_view_filters/:content_view_filter_id/rules/:id", N_("Show filter rule info")
     param :content_view_filter_id, :number, :desc => N_("filter identifier"), :required => true
     param :id, :number, :desc => N_("rule identifier"), :required => true
+    param :matching_content, :bool,
+          :desc => N_("Show content in Content View that matches the rule. Only package filters are " \
```

Comment:
Does `s/package/RPM/` make sense here? since I don't think this works for deb

Reply:
Thanks, we should probably look at updating this whole file, but I did update the places I added as you suggested :+1: